sdk/search/search-documents/package.json

```diff
@@ -89,7 +89,7 @@
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure-tools/test-recorder": "^2.0.0",
-    "@microsoft/api-extractor": "^7.31.1",
+    "@microsoft/api-extractor": "^7.33.5",
```

Comment:
nitpick: can we use `^7.31.1` to be consistent with the repo?  It makes find-and-replace easier in later upgrades.  The actual version we are getting should be the same.

Reply:
Done!